src/test/java/org/takes/rs/RsPrintTest.java

```diff
@@ -24,13 +24,21 @@
 package org.takes.rs;
 
 import java.io.IOException;
+import java.io.OutputStream;
+import org.junit.Ignore;
 import org.junit.Test;
+import org.mockito.Mockito;
 
 /**
  * Test case for {@link RsPrint}.
  * @author Yegor Bugayenko (yegor256@gmail.com)
  * @version $Id$
  * @since 0.8
+ *
+ * @todo #778:30min Fix flushHeadEvenWhenExceptionHappens test
```

Comment:
@t-izbassar I think it would be clearer to put this `@todo` directly to ignored test javadoc as described in [this post](http://www.yegor256.com/2010/03/04/pdd.html).